cranelift/zkasm_data/rem.wat

```diff
@@ -0,0 +1,14 @@
+(module
+ (import "env" "assert_eq" (func $assert_eq (param i32) (param i32)))
+ (func $main
+	i32.const 5
```

Comment:
Style nit: mixture of tabs and spaces. Pick one and stick to it. (As far as I know spaces are dominant in wasm ecosystem)